shell/platform/tizen/tizen_view_event_handler_delegate.h

```diff
@@ -0,0 +1,70 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef EMBEDDER_TIZEN_VIEW_EVENT_HANDLER_DELEGATE_H_
+#define EMBEDDER_TIZEN_VIEW_EVENT_HANDLER_DELEGATE_H_
+
+#include <memory>
```

Comment:
nit: `<memory>` doesn't seem to be used.

Reply:
I removed it.